### PR TITLE
Fix spelling errors (3)

### PR DESCRIPTION
**Pull request by docelic**
Similar to fixes done in #9844 and #9854.

This commit completes the review of all files in the repository.

### CODE REVIEW

**Review by Sija — APPROVED** [review]

**Review by straight-shoota — APPROVED** [review]
Typos in spec descriptions (for example) might not seem like the most important issues, but it's still great work. Every fixed typo is a win. Thanks for your efforts!

File: src/log/main.cr [hunk 1]
```diff
@@ -164,7 +164,7 @@ class Log
     # Log.info &.emit("Program started")                          # No data, same as Log.info { "Program started" }
     # Log.info &.emit("User logged in", user_id: 42)              # With entry data
     # Log.info &.emit(action: "Logged in", user_id: 42)           # Empty string message, only data
-    # Log.error exception: ex, &.emit("Oopps", account: {id: 42}) # With data and exception
+    # Log.error exception: ex, &.emit("Oops", account: {id: 42}) # With data and exception
```
Comment (bcardiff) [inline src/log/main.cr:167]: The other lines should get 1 space removed so # are aligned.

Comment (docelic) [reply src/log/main.cr:167]: > The other lines should get 1 space removed so # are aligned.

Indeed, done so, thanks!

**Review by jhass — APPROVED** [review]

File: src/compiler/crystal/codegen/llvm_typer.cr [hunk 2]
```diff
@@ -462,15 +462,15 @@ module Crystal
...
+    # into this typer's context.
     def copy_types(types : Array(LLVM::Type))
       types.map do |type|
         copy_type(type).as(LLVM::Type)
       end
     end
 
     # Copy an existing LLVM type, possibly from another context,
-    # into this type's context.
+    # into this typer's context.
```
Comment (jhass) [inline src/compiler/crystal/codegen/llvm_typer.cr:473]: I'm not sure this change is correct

**Pull request merged**